docker/local-metadata/docker-compose-postgres.yml

```diff
@@ -31,6 +26,8 @@ services:
       - 5432
     ports:
       - "5432:5432"
+    volumes:
+      - $PWD/docker-volume/db-data:/var/lib/postgressql
```

Comment:
Can you verify if the Github Actions CI dependent on this compose file will work as expected ?